src/view/pages/public/Home/index.js

```diff
@@ -1,3 +1,45 @@
-const Home = () => <div className='bg-primary text-white'> homepage</div>;
+import { useEffect, useState } from 'react';
+import Cog from '@heroicons/react/24/outline/CogIcon';
+
+import Api from '../../../../logic/api';
+
+const errors = (setError, err) => setError(err);
+const loading = setIsLoading => setIsLoading(false);
+const response = (setPost, res) => setPost(res?.data);
+
+const Home = () => {
+  const [post, setPost] = useState([]);
+  const [error, setError] = useState([]);
+  const [counter, setCounter] = useState(1);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    Api.get(`https://jsonplaceholder.typicode.com/posts/${counter}`)
+      .then(response.bind(null, setPost))
+      .catch(errors.bind(null, setError))
+      .finally(loading.bind(null, setIsLoading));
+  }, [counter]);
+
+  return (
+    <div className='flex flex-col h-full'>
+      <div>
+        {JSON.stringify(post)}
+        {JSON.stringify(error)}
+      </div>
+      <div className='flex flex-col justify-center items-center '>
```

Comment:
```suggestion
      <div className='flex flex-col justify-center items-center'>
```
https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

Reply:
Yes!